frontend/src/routes/index/IndexPage.vue

```diff
@@ -56,14 +57,16 @@ const incrementSize = (btnSize: number) => {
   <div :style="{ margin: '1rem', display: 'flex', alignItems: 'center' }">
     <IconButton
       :size="btn1Size"
+      :icon="vkiStarFilled"
+      color="rgb(var(--palette-yellow))"
       @click="(event: MouseEvent)=>{btn1Size = incrementSize(btn1Size)}"
```

Comment:
Čistě informativně, kontrola na 'no unused variables' tu sice není zapnutá, ale i tak je lepší to dodržovat: je zavedená konvence, že nepoužívané parametry se uvádějí s podtržítkem `_event`. No a nebo ještě lépe je neuvádět vůbec.